### ISSUE TITLE
We need distinguish persistable vars and const op

### ISSUE THREAD

**Issue by pkuyym** [PaddlePaddle/Paddle2ONNX#27]
Currently, we simply use a `Constant` operator to handle persistable variables. I think instead we should use 
``initializes`` field of onnx graph proto. Since in fluid, we have constant operators, please consider this case:
```
        cell_init = fluid.layers.fill_constant_batch_size_like(
            input=decoder_boot,
            value=0.0,
            shape=[-1, decoder_size],
            dtype='float32')
        cell_init.stop_gradient = False

        with rnn.block():
            current_word = rnn.step_input(target_embedding)
            encoder_vec = rnn.static_input(encoder_vec)
            encoder_proj = rnn.static_input(encoder_proj)
```
```cell_init``` should be the constant op which is not trainable.

**Comment by varunarora** [comment]
@pkuyym Awesome point, @kuke resolved this in #47

**Comment by pkuyym** [comment]
Resolved, colse it.
